tests/src/test/java/alluxio/client/FileOutStreamAsyncWriteIntegrationTest.java

```diff
@@ -51,13 +53,14 @@ public void asyncWriteTest() throws Exception {
     status = mFileSystem.getStatus(filePath);
     Assert.assertEquals(PersistenceState.PERSISTED.toString(), status.getPersistenceState());
 
-    checkWrite(filePath, mWriteAsync.getUnderStorageType(), length, length);
+    checkWrite(filePath, CreateFileOptions.defaults().setWriteType(WriteType.ASYNC_THROUGH).getUnderStorageType(),
```

Comment:
`CreateFileOptions.defaults().setWriteType(WriteType.ASYNC_THROUGH).getUnderStorageType()` is equivalent to `UnderStorageType.ASYNC_PERSIST`